semantic_conventions.md

```diff
@@ -52,9 +53,18 @@ Every Span log has a specific timestamp (which must fall between the start and f
 The following Span tags combine to model RPCs:
 
 - `span.kind`: either `"client"` or `"server"`. It is important to provide this tag **at Span start time**, as it may affect internal ID generation.
-- `error`: whether the RPC ended in an error
 - `peer.hostname`, `peer.ipv4`, `peer.ipv6`, `peer.port`, `peer.service`: optional tags that describe the RPC peer (often in ways it cannot assess internally)
 
+### Messaging
```

Comment:
I am really sorry to be so pedantic, but changes in this file require that :-/

"Messaging" just seems too general. I realize that I also complained that "MOM" was too obscure/overloaded.

I would vote for "Message Queues" here and either `"message_queue."` or `"mq."` as a prefix. I realize that's not a perfect fit, either, but seems like it get the point across?